### ISSUE TITLE
Random Seeding

### ISSUE THREAD

**Issue by kovasap** [peterjewicz/perlin2d#2]
When reading through your code I was thinking that this [p1 value](https://github.com/peterjewicz/perlin2d/blob/6e0cc6473a1ba460052152b11b43789c01c2187c/src/perlin2d/core.cljs#L3) has to do with the "random seed" the noise is generated from.  If this is true, it would be nice to have a docstring on it describing that and also explaining why it is such a long vector as opposed to a single number.  Maybe we should add a function that generates this from a single number?

**Comment by peterjewicz** [comment]
@kovasap That's correct, it's used for the seeding value. I agree that it could use some documentation on why it's there and why it's the length it is. 

On the seeding, I think that would be a cool addition to be able to pass a single number to generate it from. I originally wrote this just to be completely random, but if there's some interest in specific seeding I can work on that. I should have some time over the weekend into next week to take a look at adding that. Thanks for the suggestions!

**Comment by kovasap** [comment]
Awesome!  I did notice interestingly in my clojurescript app that my noise does end up looking different whenever I reload the page.  Is this expected or suggestive of a bug somewhere (probably in my code)?

**Comment by peterjewicz** [comment]
That's intended given the current implementation. Line 18 shuffles the seed vector, so it should always be different leading to different noise on each call.
